src/Command/BackupList.php

```diff
@@ -15,6 +15,8 @@
 
 /**
  * CLI command for showing the list of backup files.
+ *
+ * @api
```

Comment:
Why do we mark CLI command as API? 
@shiftedreality , @oshmyheliuk

Reply:
We decided to mark them as `@api`  to show that their call/result won't be changed.
I agree that command Class is not `@api` It can be documented in dev docs instead of marking command classes.

Reply:
Having annotation is still more clear than DevDocs, additionally we’ll have them explained in docs

Reply:
I think that for such cases like with CLI commands we should have separate tag, but we do not have in phpdoc. 
@oshmyheliuk ok, lets leave it